### ISSUE TITLE
Can't disenchant BE icons

### ISSUE THREAD

**Issue by marvinscham** [marvinscham/disenchanter#128]
**Description (if necessary)**
Summoner icons that disenchant into Blue Essence (instead of the usual Orange Essence) such as Champie icons will silently fail to disenchant.

**Version**
1.5.0

**Comment by marvinscham** [comment]
Recipes of Elise Champie Icon:
```json
[
  {
    "contextMenuText": "",
    "crafterName": "disenchant",
    "description": "",
    "displayCategories": "SUMMONERICON",
    "imagePath": "",
    "introVideoPath": "",
    "loopVideoPath": "",
    "lootMilestoneIds": [],
    "metadata": {
      "bonusDescriptions": [],
      "guaranteedDescriptions": [],
      "tooltipsDisabled": false
    },
    "outputs": [{ "lootName": "CURRENCY_champion", "quantity": 2500 }],
    "outroVideoPath": "",
    "recipeName": "SUMMONERICON_Champie_disenchant",
    "requirementText": "",
    "slots": [
      {
        "lootIds": ["SUMMONER_ICON_5958"],
        "quantity": 1,
        "slotNumber": 0,
        "tags": "nodropoutput,champie,noreroll,norerolloutput,shadowisles"
      }
    ],
    "type": "DISENCHANT"
  }
]
```

**Comment by marvinscham** [comment]
Fixed in #143
